FoodOrderingSystem.py

```diff
@@ -0,0 +1,700 @@
+import time
+import datetime
+
+from datetime import datetime, timedelta
+
+from sqlalchemy import Column, String, Integer, DateTime, ForeignKey
+from sqlalchemy import create_engine, func, update
+
+from sqlalchemy.sql import func
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+class FoodCategory(Base):
+    """ Represents food categories """
+    __tablename__ = 'food_category'
+
+    category_id = Column(Integer(), primary_key=True, nullable=False, unique=True, autoincrement=True)
+    name = Column(String(30), nullable=False)
+
+class FoodDetails(Base):
+    """ Represents food details """
+    __tablename__ = 'food_details'
+
+    food_id = Column(Integer(), primary_key=True, nullable=False, unique=True, autoincrement=True)
+    category_id = Column(Integer(), ForeignKey('food_category.category_id'), nullable=False)
+    food_name = Column(String(30), nullable=False)
+    price = Column(Integer(), nullable=False)
+
+class CustomerDetails(Base):
+    """ Represents customer details """
+    __tablename__ = 'customer_details'
+
+    cust_id = Column(Integer(), primary_key=True, nullable=False, unique=True, autoincrement=True)
+    cust_name = Column(String(30), nullable=False)
+    cust_phone = Column(Integer())
+    cust_email = Column(String(30))
+
+class CustOrderSelection(Base):    
+    """ Represents food ordered """
+    __tablename__ = 'customer_order_selection'
+
+    order_id = Column(Integer(), ForeignKey('customer_order_status.order_id'), primary_key=True, nullable=False)
+    food_id = Column(Integer(), ForeignKey('food_details.food_id')) 
+    food_qty = Column(Integer())
```

Comment:
I don't get what's food_qty in CustOrderSelection? Please explain.

Reply:
food_qty is when a customer wants to order multiple food items.

Eg: Customer 1 wants to order an appetizers for him and another person.

Therefore quantity is 2.